spec/models/mapit_postcode_response_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "spec_helper"
+
+RSpec.describe MapitPostcodeResponse do
+  let(:location) do
+    {
+      "codes" => {
+        "ons" => "01",
+        "gss" => "E01000123",
+        "govuk_slug" => "test-one",
+      },
+      "name" => "Coruscant Planetary Council",
+      "type" => "LBO",
+      "country_name" => "England",
+    }
+  end
+
+  it "returns the gss code" do
+    expect(described_class.new(location).gss).to eq("E01000123")
+  end
+
+  it "returns the location name" do
```

Comment:
could reword to 'returns the area name' ?

Reply:
Updated and fixed up.